focus_converter_base/focus_converter/conversion_configs/azure/D021_S001.yaml

```diff
@@ -0,0 +1,10 @@
+plan_name: convert Frequency to ChargeFrequency
+conversion_type: sql_condition
+conversion_args:
+    conditions:
+        - WHEN Frequency = 'OneTime' THEN 'One-Time'
+        - WHEN Frequency = 'Recurring' THEN 'Recurring'
+        - WHEN Frequency = 'UsageBased' THEN 'Usage-Based'
+    default_value: "Other"
```

Comment:
default_value should be 'Other' with single quote otherwise it looks for a column called "Other"

Reply:
I think I found a bug in my implementation and need to fix the SQL condition.

Reply:
Blocked by #294.

Reply:
@csyvenky-finops sorry again, default_value needs to be ``Other``.

single quote single quote Other single quote single quote.

Reply:
@csyvenky-finops, I am sorry I was trying to add a review commit but it ended up writing one commit on top.

We needed it to be enclosed in double_quotes:
default_value: "'Other'"

If you think this works let me know and we can merge this.